README.md

```diff
@@ -65,46 +44,26 @@ installed run
 
 ### Interacting with the Demo
 
-## CLI
-
-We have built a simple CLI to interact with the demo using a few preseeded identities (Alice, Bob, Charlie). With the
-CLI, you can perform asset transfers and check balances. With the demo running, open another terminal and try
-transferring some dummy tokens:
-
-```console
-> scripts/cli check-balance Alice
-Balance of 0x885ee92eebda03540066a25a57cc625bbee15d5a: 9999
-
-> scripts/cli transfer Alice Bob 1000
-Submitting Transaction to Rollup API: Transferring 1000 tokens from 0x885e…5d5a to 0xf236…69ca
-
-> scripts/cli check-balance Alice
-Balance of 0x885ee92eebda03540066a25a57cc625bbee15d5a: 8999
-```
-
-Run `scripts/cli --help` for more information. If you are running the demo natively replace `scripts/cli` with
-`just cli`.
-
 ## Curl
 
 With the demo running, navigate to http://localhost:8082/ for API documentation.
 
 1. Query the genesis account balance:
 
 ```
-curl http://localhost:8082/rollup/balance/0xf23694f9c6d4837fc596c4eb7c3c3d8a8bae69ca
+curl http://localhost:8084/v0/rollup/balance/0xf23694f9c6d4837fc596c4eb7c3c3d8a8bae69ca
 ```
 
 2. Send tokens to `0x885ee92eebda03540066a25a57cc625bbee15d5a`:
 
 ```
-curl -X POST -H "Content-Type: application/json" http://localhost:8082/rollup/submit -d "{\"transaction\":{\"amount\":100,\"destination\":\"0x885ee92eebda03540066a25a57cc625bbee15d5a\",\"nonce\":1},\"signature\":{\"r\":\"0x61395b25cf41321bc1242ec301c0aa5a5e5ff47b697f80119a20ce3e5be66f9e\",\"s\":\"0x447cf03a5ddb28b9a189d108a8e91efa523fd3fb37cebab1cad610d82a8edbb0\",\"v\":27}}"
+curl -X POST -H "Content-Type: application/json" http://0.0.0.0:8084/v0/rollup/submit -d "{\"transaction\":{\"amount\":100,\"destination\":\"0x885ee92eebda03540066a25a57cc625bbee15d5a\",\"nonce\":1},\"signature\":{\"r\":\"0x61395b25cf41321bc1242ec301c0aa5a5e5ff47b697f80119a20ce3e5be66f9e\",\"s\":\"0x447cf03a5ddb28b9a189d108a8e91efa523fd3fb37cebab1cad610d82a8edbb0\",\"v\":27}}"
 ```
 
 3. Query `0x885ee92eebda03540066a25a57cc625bbee15d5a` balance:
 
 ```
-curl http://localhost:8082/rollup/balance/0x885ee92eebda03540066a25a57cc625bbee15d5a
+curl http://localhost:8084/v0/rollup/balance/0x885ee92eebda03540066a25a57cc625bbee15d5a
```

Comment:
I got this running! Woohoo!

When sending the tokens (in step 2 above) and then curling to get the balance - the balance didn't change

Reply:
Nice!!

did you wait for `Applied transaction 1 for 0xf236…69ca and Proof was submitted successfully` It sometimes takes time to come and then the balance updates after a few blocks?